src/extension/extensionServer.ts

```diff
@@ -123,6 +124,19 @@ export class ExtensionServer implements vscode.Disposable {
         return Q.resolve<void>(void 0);
     }
 
+    /**
+     * Message handler for OPEN_FILE_AT_LOCATION
+     */
+    private openFileAtLocation(filename: string, lineNumber: number): Q.Promise<void> {
+        return Q(vscode.workspace.openTextDocument(vscode.Uri.file(filename)).then(document => {
+            return vscode.window.showTextDocument(document).then(editor => {
+                let range = editor.document.lineAt(lineNumber - 1).range;
+                editor.selection = new vscode.Selection(range.start, range.end);
+                editor.revealRange(range, vscode.TextEditorRevealType.InCenter);
```

Comment:
It will actually select the whole line. This is the current behavior of vscode if you use the command `code workspace -g file:number`